ci/test.sh

```diff
@@ -39,6 +39,11 @@ done
 ./tools/install_config.sh --config=config/config_dev.json --update
 ./tools/setup_local_database.sh --nopassword --db_user ubuntu --db_name circle_test
 
+# Verify that data generation works.
+./tools/generate_fake_data.sh \
+    --num_participants 10 \
```

Comment:
This may slow things down a bit. Maybe that's fine. Just wondering if we should limit it to 1 participant.

Reply:
On a local setup (as in CircleCI) 10 only takes a couple seconds, but may as well shorten it. I went with 3 just so it's still multiple (I'm not sure if there's logic that's different between 1 and >1, but there might be).